From: Iain Lane <iain.lane@canonical.com>
Description: Fix arm64 build
 Add AArch64 support in Platform.h
 Don't use COMPARE_AND_SWAP on arm64; it produces invalid asm.
 Fix some other areas where disabling JIT doesn't work.
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=752735
Forwarded: no
Taken from: https://sources.debian.org/patches/openjfx/11.0.2+1-1/fix-arm64-build.patch
--- old/modules/javafx.web/src/main/native/Source/WTF/wtf/Platform.h
+++ new/modules/javafx.web/src/main/native/Source/WTF/wtf/Platform.h
@@ -146,7 +146,9 @@
 #if (defined(__arm64__) && defined(__APPLE__)) || defined(__aarch64__)
 #define WTF_CPU_ARM64 1
 #define WTF_CPU_KNOWN 1
-
+#if defined(__AARCH64BE__)
+#define WTF_CPU_BIG_ENDIAN 1
+#endif
 #if defined(__arm64e__)
 #define WTF_CPU_ARM64E 1
 #endif
